src/olympia/blocklist/tests/test_commands.py

```diff
@@ -64,11 +71,21 @@ def test_regex(self):
             assert block.kinto_id == '*' + this_block['id']
             assert block.include_in_legacy
             assert block.modified == datetime(2019, 11, 29, 22, 22, 46, 785000)
+        assert KintoImport.objects.count() == 6
+        assert KintoImport.objects.filter(
+            outcome=KintoImport.OUTCOME_NOMATCH).count() == 4
+        assert KintoImport.objects.filter(
+            outcome=KintoImport.OUTCOME_NOTFIREFOX).count() == 1
```

Comment:
If there were a way to configure the `blocklist.json` in the test, as mentioned above, then you wouldn't need to assert this in each test, where the tests are not about testing this at all.